charts/nexus3/templates/deployment.yaml

```diff
@@ -150,6 +150,12 @@ spec:
               name: {{ include "nexus3.fullname" . }}-properties
               subPath: nexus.properties
               readOnly: true
+          {{- if .Values.license.enabled}}
+            - mountPath: /nexus-data/etc/nexus.license
+              name: {{ include "nexus3.fullname" . }}-license
+              subPath: nexus.license
+              readOnly: true
+          {{- end }}
```

Comment:
```suggestion
```

Reply:
The `Deployment` implementation is legacy functionality so I don't add new features there.

Reply:
Ok, shouldn't that be mentioned in the README?

Reply:
This is actually the one original chart not to get a SemVer major bump to deprecate the `Deployment` pattern. I'll add something to the README before release to cover recommending use as a `StatefulSet`.